test/Microsoft.Azure.ServiceBus.UnitTests/QueueClientTests.cs

```diff
@@ -174,7 +174,7 @@ public sealed class QueueClientTests : SenderReceiverClientTestBase
             var queueClient = (QueueClient)messagingFactory.CreateQueueClientFromConnectionString(TestUtility.GetEntityConnectionString(queueName), ReceiveMode.ReceiveAndDelete);
             try
             {
-                await this.ReceiveShouldReturnNoLaterThanServerWaitTimeTestCase(queueClient.InnerSender, queueClient.InnerReceiver, messageCount);
+                await this.ReceiveShouldReturnNoLaterThanServerWaitTimeTestCase(queueClient.InnerReceiver);
```

Comment:
Based on previous comment, remove the 'InnerReceiver' .
Same for all other occurrences